pkg/cmd/workspace/create/creation_data.go

```diff
@@ -227,12 +227,18 @@ func GetBranchFromWorkspaceTemplate(ctx context.Context, workspaceTemplate *apic
 		return nil, apiclient_util.HandleErrorResponse(res, err)
 	}
 
+	gitProvider, res, err := apiClient.GitProviderAPI.GetGitProvider(ctx, gitProviderConfigId).Execute()
```

Comment:
![Screenshot 2024-12-12 at 14 42 01](https://github.com/user-attachments/assets/a6d51fa5-fcd5-4169-9d02-962a514a5048)

This is the error I get when running a build from a WT while not having any git provider registered